examples/contour/01-contour-config-crd.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: projectcontour.io/v1alpha1
```

Comment:
I have this just to play around with applying various configs + testing defaults. We'll need to clean this up and come up with a valid "default".

Reply:
aaaand I took it out. Was causing all kinds of issues. =)